resources/js/components/ui/avatar.tsx

```
@@ -36,7 +36,7 @@ export function Avatar({
       data-slot="avatar"
       {...props}
       className={twMerge(
-        "-outline-offset-1 inline-grid size-(--avatar-size) shrink-0 align-middle outline-1 outline-fg/(--ring-opacity) [--avatar-radius:20%] [--ring-opacity:20%] *:col-start-1 *:row-start-1 *:size-(--avatar-size)",
+        "-outline-offset-1 inline-grid size-(--avatar-size) shrink-0 align-middle outline-1 outline-foreground/(--ring-opacity) [--avatar-radius:20%] [--ring-opacity:20%] *:col-start-1 *:row-start-1 *:size-(--avatar-size)",
```

Biome (2.4.13)

[error] These CSS classes should be sorted.

(lint/nursery/useSortedClasses)